.github/workflows/pa11yci.yml

```diff
@@ -15,8 +19,8 @@ jobs:
     - name: Set up Node.js
       uses: actions/setup-node@v2
       with:
-        node-version: 20.14.0
-
+        node-version: 20.x
+    
```

Comment:
is this needed?

Reply:
This is just to make sure that we are using the latest version of node. Would it be preferred if we just kept it as a hardcoded version like we have with the linter.yml? (20.14.0)